Tests are failing locally

@Stebalien is it me or is it the code?

```
~/.g/s/g/i/go-ipld-cbor ❯❯❯ git rev-parse HEAD                                                                               ✘ 1 master
b7e8f2be4f3ecd5a9616cfd311c3165cb55efba3
~/.g/s/g/i/go-ipld-cbor ❯❯❯ gx test                                                                                              master
cid: QmSdenoaYqAXRnTykZAtvRosPX8vQFFXCnsgiAwXJ8eLSW
before [162 99 98 97 114 216 42 88 35 0 18 32 63 201 182 137 69 157 115 143 140 136 163 164 138 169 227 53 66 1 107 122 64 82 224 1 170 165 54 252 167 72 19 203 100 110 97 109 101 99 102 111 111]
after [162 99 98 97 114 216 42 88 35 0 18 32 63 201 182 137 69 157 115 143 140 136 163 164 138 169 227 53 66 1 107 122 64 82 224 1 170 165 54 252 167 72 19 203 100 110 97 109 101 99 102 111 111]
--- FAIL: TestBigIntRoundtrip (0.00s)
    node_test.go:556: missing an atlas entry describing how to marshal type big.Int (and auto-atlasing for structs is not enabled)
FAIL
exit status 1
FAIL	github.com/ipfs/go-ipld-cbor	0.008s
ERROR: Error: exit status 1
ERROR: test hook failed: exit status 1
```

--- a/node_test.go
+++ b/node_test.go
@@ -512,6 +512,7 @@ func TestCanonicalStructEncoding(t *testing.T) {
 		Whale string
 		Cat   bool
 	}
+	RegisterCborType(BigIntAtlasEntry)
 	RegisterCborType(Foo{})
 
 	s := Foo{

